src/test/java/io/jenkins/plugins/credentials/secretsmanager/FiltersIT.java

```diff
@@ -19,11 +18,15 @@
     public void shouldFilterByTag() {
         // Given
         final CreateSecretOperation.Result foo = createSecret("supersecret", opts -> {
-            opts.tags = Collections.singletonMap("product", "roadrunner");
+            opts.tags = Maps.of(
+                    "jenkins:credentials:type", "string",
+                    "product", "roadrunner");
         });
         // And
         final CreateSecretOperation.Result bar = createOtherSecret("supersecret", opts -> {
-            opts.tags = Collections.singletonMap("product", "coyote");
+            opts.tags = Maps.of(
+                    "jenkins:credentials:type", "string",
```

Comment:
Can we create a static string to hold this value and then just reference it here?  A good spot for this static string may be the `CredentialsFactory` class.

Reply:
Good idea, I've formalised two classes to contain the constants - `Tags` and `Type`.